src/containers/pages/FocusInvestigation/map/active/tests/supersetFixtures/jurisdictionsNoGeojson.json

```diff
@@ -0,0 +1,328 @@
+{
+  "cache_key": null,
+  "cached_dttm": null,
+  "cache_timeout": 300,
+  "error": null,
+  "form_data": {
+    "datasource": "28__table",
+    "viz_type": "table",
+    "granularity_sqla": null,
+    "time_grain_sqla": "P1D",
+    "time_range": "100 years ago : ",
+    "groupby": [],
+    "metrics": [],
+    "percent_metrics": [],
+    "timeseries_limit_metric": null,
+    "row_limit": 10000,
+    "include_time": false,
+    "order_desc": true,
+    "all_columns": ["jurisdiction_id", "geojson"],
+    "order_by_cols": [],
+    "adhoc_filters": [
+      {
+        "clause": "WHERE",
+        "expressionType": "SIMPLE",
+        "comparator": "3951",
+        "operator": "==",
+        "subject": "jurisdiction_id"
+      }
+    ],
+    "table_timestamp_format": "%Y-%m-%d %H:%M:%S",
+    "page_length": 0,
+    "include_search": false,
+    "table_filter": false,
+    "align_pn": false,
+    "color_pn": true,
+    "slice_id": 459,
+    "where": "",
+    "having": "",
+    "having_filters": [],
+    "filters": [{ "col": "jurisdiction_id", "op": "==", "val": "3951" }]
+  },
+  "is_cached": false,
+  "query": "SELECT jurisdiction_id AS jurisdiction_id,\n       geojson AS geojson\nFROM\n  (SELECT jurisdiction_id,\n          jsonb_build_object('type', 'Feature', 'id', jurisdiction_id, 'geometry', ST_AsGeoJSON(jurisdiction_geometry)::jsonb, 'properties', to_jsonb(row) - 'jurisdiction_id' - 'jurisdiction_geometry') AS geojson\n   FROM\n     (SELECT jurisdiction_id,\n             jurisdiction_name,\n             jurisdiction_parent_id,\n             jurisdiction_geometry\n      FROM plans_materialzied_view) row) AS expr_qry\nWHERE jurisdiction_id = '3951'\nLIMIT 10000;",
+  "status": "success",
+  "stacktrace": null,
+  "rowcount": 92,
+  "data": {
+    "records": [
+      {
+        "jurisdiction_id": "3951"
+      },
+      {
+        "jurisdiction_id": "3951"
+      },
+      {
+        "jurisdiction_id": "3951"
+      },
+      {
+        "jurisdiction_id": "3951"
+      },
+      {
+        "jurisdiction_id": "3951"
+      },
+      {
```

Comment:
Why are these repeated this way?

Reply:
Its also repeated in the other fixture, but I've refactored to only pass the array